test-karate/src/test/java/graphql/ftm/helpers.feature

```diff
@@ -0,0 +1,120 @@
+Feature: Reusable Functions and Helpers
+
+  Scenario: Define reusable functions
+
+    ## Builds a payload for creating a new content version
+    * def buildContentRequestPayload =
+      """
+      function(contentType, title, publishDate, expiresOn, identifier) {
+        var payload = {
+          "contentlets": [
+            {
+              "contentType": contentType,
+              "title": title,
+              "host":"8a7d5e23-da1e-420a-b4f0-471e7da8ea2d"
+            }
+          ]
+        };
```

Comment:
If this is JS, avoid the use of `var` those kind of variables sometimes are initialized as global and they get hoisted to the top of the file leading to errors that are hard to identify and even overwritting other vars/functions

Reply:
The recommended way is using [let](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/let) or [const](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Statements/const) to maintain the variables in the expected scope

Reply:
done